sdv/single_table/base.py

```diff
@@ -270,6 +270,44 @@ def get_metadata(self):
         """Return the ``SingleTableMetadata`` for this synthesizer."""
         return self.metadata
 
+    def auto_assign_transformers(self, data):
+        """Automatically assign the required transformers for the given data and constraints.
+
+        This method will automatically set a configuration to the ``rdt.HyperTransformer``
+        with the required transformers for the current data.
+
+        Args:
+            data (pandas.DataFrame):
+                The raw data (before any transformations) that will be used to fit the model.
+        """
+        constrained = self._data_processor._fit_transform_constraints(data)
+        columns_created_by_constraints = set(constrained.columns) - set(data.columns)
+        self._data_processor._fit_numerical_formatters(data)
+        config = self._data_processor._create_config(constrained, columns_created_by_constraints)
+        self._data_processor._hyper_transformer.set_config(config)
+
+    def get_transformers(self):
+        """Get a dictionary mapping of ``column_name``  and ``rdt.transformers``.
+
+        A dictionary representing the column names and the transformers that will be used
+        to transform the data.
+
+        Returns:
+            dict:
+                A dictionary mapping with column names and transformers.
+        """
+        field_transformers = {}
```

Comment:
The logic can be tidied up a little:

```python3
if self._data_processor._hyper_transformer is not None:
    ft = self._data_processor._hyper_transformer.field_transformers
    if ft: return ft
raise
```

Reply:
Actually I forgot to change this logic, as `hyper_transformer` will no longer be `None`!